bin/rtfm

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+require 'richmond'
+
+Richmond.generate ARGV[0]
```

Comment:
I'd recommend throwing in some sort of executable.  This is an example of how it works.

So I'd install your gem, and then I'd make a statement like

```ruby
rtfm /home/darrencauthon/my_gem
```

Or however else you'd want to do it.